src/extensions/vb/Microsoft.DotNet.UpgradeAssistant.Extensions.VisualBasic/MyDotAnalyzer.cs

```diff
@@ -70,7 +70,7 @@ public async Task AnalyzeAsync(IProject project, IDependencyAnalysisState state,
                     // 1>C:\{ProjectDir}\My Project\Settings.Designer.vb(57,10): error BC30002: Type 'Global.System.Configuration.UserScopedSettingAttribute' is not defined.
                     // 1>C:\{ProjectDir}\My Project\Settings.Designer.vb(59,10): error BC30002: Type 'Global.System.Configuration.DefaultSettingValueAttribute' is not defined.
                     _logger.LogInformation("Reference to configuration package ({SystemConfigurationPackageName}, version {SystemConfigurationPackageVersion}) needs added", SystemConfigurationPackageName, systemConfigurationPackage.Version);
-                    state.Packages.Add(systemConfigurationPackage);
+                    state.Packages.Add(systemConfigurationPackage, new());
```

Comment:
You can remove this with the support kept for BuildBreakRisk

Reply:
removing throws a warning, so made the call explicit.